.github/workflows/python-test-library.yml

```
@@ -0,0 +1,42 @@
+name: Python Tests for Libraries
+
+on:
+  workflow_call:
+    inputs:
+      PYTHON_VERSION:
+        required: true
+        type: string
+        description: The python version to test and build against
+  workflow_dispatch:
+
+jobs:
+  test_and_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ inputs.PYTHON_VERSION }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ inputs.PYTHON_VERSION }}
+
+      - name: Prepare pip and and pip-tools
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pip-tools
+
+      - name: Generate Requirements without Code Artifact
```

```suggestion
      - name: Generate Requirements
```
No need to reference Code Artifact anymore